weekly-updates/weekly-update.2015-10-16.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Weekly Update - Oct 16th, 2015
+author: Minwoo Jung (@jmwsoft)
+date: 2015-10-16T12:00:00.000Z
+status: publish
+category: weekly
+slug: weekly-update-2015-10-16
+layout: blog-post.hbs
+---
+
+### Node.js News — October 16th
+Node.js v4.2.1 (LTS), v4.2.0 (LTS) are released
+
+### Node.js v4.2.1 (LTS) and v4.2.0 (LTS) Releases
+
+This week we have two releases: [Node.js v4.2.1 (LTS)](https://nodejs.org/en/blog/release/v4.2.1/) and [Node.js v4.2.0 (LTS)](https://nodejs.org/en/blog/release/v4.2.0/). complete changelog from previous releases can be found [on GitHub](https://github.com/nodejs/node/blob/master/CHANGELOG.md).
+
+### Notable changes
+
+* Includes fixes for two regressions
+  - Assertion error in WeakCallback  - see [#3329](https://github.com/nodejs/node/pull/3329)
+  - Undefined timeout regression - see [#3331](https://github.com/nodejs/node/pull/3331)
+
+### Known issues
```

Comment:
Should we just `Known Issues` in the weekly update? Seems like a lot of detail that most people probably skip over. If it's already in the linked announcements, that's probably enough.